_includes/v20.2/prod-deployment/insecure-start-nodes.md

```diff
@@ -24,6 +24,35 @@ After completing these steps, nodes will not yet be live. They will complete the
     | tar  xvz
     ~~~
 
+echo 'Installing CockroachDB v20.2 and supporting spatial libraries...'
```

Comment:
Looking at the [CI-built page](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/fe1f98db2c245962f185cd3d67ee5ec5f4f98246/stable/deploy-cockroachdb-on-premises-insecure.html) it looks like this section is not rendered in a code block.  But it seems like it's supposed to be.  Is that correct?